chatkit-core/src/test/kotlin/com/pusher/chatkit/users/UserApiTypeMapperTest.kt

```diff
@@ -0,0 +1,76 @@
+package com.pusher.chatkit.users
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isNull
+import com.pusher.chatkit.users.api.UserApiType
+import com.pusher.chatkit.users.api.UserApiTypeMapper
+import com.pusher.chatkit.util.DateApiTypeMapper
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class UserApiTypeMapperTest : Spek({
+    describe("given a complete UserApiType") {
+        val userApiType = UserApiType(
+            id = "alice",
+            createdAt = "2020-03-05T15:31:21Z",
+            name = "alice",
+            customData = mapOf("item" to "data"),
+            avatarUrl = "http://placekitten.com/200/200",
+            updatedAt = "2020-03-05T15:31:42Z"
+        )
+        describe("when mapped to a UserInternalType object") {
+            val dateApiTypeMapper = DateApiTypeMapper()
+            val userInternalType = UserApiTypeMapper(dateApiTypeMapper)
+                .toUserInternalType(userApiType)
+
+            it("then the id matches") {
+                assertThat(userInternalType.id).isEqualTo(userApiType.id)
+            }
+
+            it("then the name matches") {
+                assertThat(userInternalType.name).isEqualTo(userApiType.name)
+            }
+
+            it("then the customData matches") {
```

Comment:
```suggestion
            it("then customData matches") {
```
[consistency](https://github.com/pusher/chatkit-android/blob/e8faf448dd3ccf4bca3b620f7425afa332e3a4bb/chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/api/JoinedRoomApiTypeMapperTest.kt#L43) (applies to other places too)

Reply:
Have removed all the `the`'s from this file.